src/libraries/System.Net.Sockets/tests/FunctionalTests/SocketDuplicationTests.cs

```diff
@@ -27,15 +27,6 @@ public class SocketDuplicationTests
         private static string GetMessageString(ArraySegment<byte> data, int count) =>
             Encoding.ASCII.GetString(data.AsSpan(0, count));
 
-        [Fact]
-        public void UseOnlyOverlappedIO_AlwaysFalse()
-        {
-            using Socket s = new Socket(AddressFamily.InterNetwork, SocketType.Stream, ProtocolType.Tcp);
-
-            Assert.False(s.UseOnlyOverlappedIO);
-            s.UseOnlyOverlappedIO = true;
```

Comment:
We need to keep some of the tests around, with the warning disabled around the use in tests.

We need to make sure that it still works as expected even once it is obsolete.

Reply:
How can I add `pragma warning disable` without diagnstic id?

Reply:
Oh, I can use CS0618
Okay, I'll return tests 👍

Reply:
https://github.com/dotnet/runtime/search?q=0618